### ISSUE TITLE
Second `md.pattern()` call does not render plot correctly

### ISSUE THREAD

**Issue by hanneoberman** [amices/mice#318]
The plot of the missing data pattern is cropped, see reprex below. 

``` r
library(mice)
#> Warning: package 'mice' was built under R version 4.0.4
#> 
#> Attaching package: 'mice'
#> The following object is masked from 'package:stats':
#> 
#>     filter
#> The following objects are masked from 'package:base':
#> 
#>     cbind, rbind
md.pattern(boys)
#>     age reg wgt hgt bmi hc gen phb  tv     
#> 223   1   1   1   1   1  1   1   1   1    0
#> 19    1   1   1   1   1  1   1   1   0    1
#> 1     1   1   1   1   1  1   1   0   1    1
#> 1     1   1   1   1   1  1   0   1   0    2
#> 437   1   1   1   1   1  1   0   0   0    3
#> 43    1   1   1   1   1  0   0   0   0    4
#> 16    1   1   1   0   0  1   0   0   0    5
#> 1     1   1   1   0   0  0   0   0   0    6
#> 1     1   1   0   1   0  1   0   0   0    5
#> 1     1   1   0   0   0  1   1   1   1    3
#> 1     1   1   0   0   0  0   1   1   1    4
#> 1     1   1   0   0   0  0   0   0   0    7
#> 3     1   0   1   1   1  1   0   0   0    4
#>       0   3   4  20  21 46 503 503 522 1622
md.pattern(boys)
```

![](https://i.imgur.com/CpaMs4e.png)

    #>     age reg wgt hgt bmi hc gen phb  tv     
    #> 223   1   1   1   1   1  1   1   1   1    0
    #> 19    1   1   1   1   1  1   1   1   0    1
    #> 1     1   1   1   1   1  1   1   0   1    1
    #> 1     1   1   1   1   1  1   0   1   0    2
    #> 437   1   1   1   1   1  1   0   0   0    3
    #> 43    1   1   1   1   1  0   0   0   0    4
    #> 16    1   1   1   0   0  1   0   0   0    5
    #> 1     1   1   1   0   0  0   0   0   0    6
    #> 1     1   1   0   1   0  1   0   0   0    5
    #> 1     1   1   0   0   0  1   1   1   1    3
    #> 1     1   1   0   0   0  0   1   1   1    4
    #> 1     1   1   0   0   0  0   0   0   0    7
    #> 3     1   0   1   1   1  1   0   0   0    4
    #>       0   3   4  20  21 46 503 503 522 1622

<sup>Created on 2021-03-15 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

**Comment by stefvanbuuren** [comment]
Hmm, that's curious. In Studio 1.4.1032 on MacOS I get the full plot.

**Comment by stefvanbuuren** [comment]
Ah, I now see what you mean. 
It's the second invocation of the function. Perhaps `md.pattern()` changes `par()` and does not restore.

**Comment by gerkovink** [comment]
I believe this is solved pending PR #323
